packages/thirdweb/src/react/hooks/pay/useSendSwap.ts

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { useSwapStatus, type SwapStatusParams } from "./useSwapStatus.js";
+import { useMutation } from "@tanstack/react-query";
+import { useActiveAccount } from "../../providers/wallet-provider.js";
+import { type SwapRoute } from "../../../pay/swap/actions/getSwap.js";
```

Comment:
```suggestion
import type{ SwapRoute } from "../../../pay/swap/actions/getSwap.js";
```

otherwise bundlers will always import a little bit of the file because they can't fully eliminate the import for runtime

Reply:
same in other places where all you are importing from a file is types, it's fine to mix the `type` in the named import when you're also importing runtime code